storage/sql/crud.go

```diff
@@ -516,6 +516,32 @@ func (c *conn) UpdateClient(id string, updater func(old storage.Client) (storage
 	})
 }
 
+func (c *conn) UpdateBlockedUser(username string, updater func(old storage.BlockedUser) (storage.BlockedUser, error)) error {
```

Comment:
instead of UpdateBlockedUser & CreateBlockedUser can't we write one method for upsert?

Reply:
We can, but in terms of code readability update and create looks correct.